src/components/PhotoCard/PhotoCard.module.scss

```diff
@@ -44,8 +38,7 @@ $black: #000000;
         background-image: url('../../static/defaults/placeholder.png');
         background-size: contain;
         cursor: pointer;
-        margin-right: 10px;
-        flex-shrink: 0;
```

Comment:
flex-shrink is needed here to ensure that this doesn't happen to the picture like it has here: 
![Screenshot 2019-10-07 at 4 27 44 pm](https://user-images.githubusercontent.com/45773586/66325997-275ad400-e920-11e9-89b3-362fde7c4b21.png)

Reply:
Ah ok